zkevm-circuits/src/util.rs

```diff
@@ -54,7 +54,7 @@ impl<F: FieldExt> Expr<F> for i32 {
     #[inline]
     fn expr(&self) -> Expression<F> {
         Expression::Constant(
-            F::from(self.abs() as u64)
+            F::from(self.unsigned_abs() as u64)
```

Comment:
I still prefer `unsigned_abs` TBH. I know this should never be an issue. But since is the same, let's be sure we always get uints